src/admin/dto/input/login.input.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, InputType } from '@nestjs/graphql';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+@InputType()
+export class LoginInput {
+  // email validation should be added in future
```

Comment:
this is username field, why email validation should be added?

Reply:
nope, as I saw in the slack [discussion](https://fullstacksjs.slack.com/archives/C03RY94DULS/p1665676791560889) @ASafaeirad said the username should be a valid email address.

Reply:
Aha, I got you.

Reply:
but why you just didn't add that and add a TODO comment?

Reply:
done

Reply:
you get me wrong on this.
I mean why you just didn't add `@IsEmail` decorator?

Reply:
yeap it was a good question, because now there is a username field :))